theworkoutinfo/ViewModels/HomeViewModel.swift

```diff
@@ -9,32 +9,43 @@
 import Foundation
 import RxSwift
 
+public typealias Animate = (wantAnimate: Bool, message: String?)
+
 class HomeViewModel {
     
     let disposeBag = DisposeBag()
     
     //MARK: - Inputs
     
     let tryLogout: AnyObserver<Void>
-    let loadExercises: AnyObserver<Int>
+    let loadExercises: AnyObserver<(Bool,Int)>
     let selectExercise: AnyObserver<Exercise>
+    let loadExercisesByCategory: AnyObserver<(Bool,ExerciseCategory,Int)>
+    let loadCategories: AnyObserver<Void>
+    let tryAnimate: AnyObserver<Animate>
```

Comment:
tryAnimate should not be public, since it will be handled internally in the viewModel and not from outside. In fact it doesn't even have to be a class var, it can be defined in the init method